### ISSUE TITLE
Separate windows don't have enough border contrast when on top of each other in high contrast mode

### ISSUE THREAD

**Issue by mousetraps** [Microsoft/vscode#19309]
See below:
![image](https://cloud.githubusercontent.com/assets/762848/22272841/9ce64566-e251-11e6-9b04-bfbfc7f64d34.png)

**Comment by bpasero** [comment]
This should be something that we can control via window options natively ideally and not via some CSS border we add around the body element.

**Comment by bpasero** [comment]
Closing, as I have not heard other complaints so far.
